conf/lsf.conf

```diff
@@ -59,6 +59,20 @@ backend {
         """
 
         submit-docker = """
+          # Make sure the SINGULARITY_CACHEDIR variable is set. If not use a default
+          # based on the users home.
+          if [ -z $SINGULARITY_CACHEDIR ];
+              then CACHE_DIR=$HOME/.singularity/cache
+              else CACHE_DIR=$SINGULARITY_CACHEDIR
+          fi
+          # Make sure cache dir exists so lock file can be created by flock
+          mkdir -p $CACHE_DIR
+          LOCK_FILE=$CACHE_DIR/singularity_pull_flock
+          # Create an exclusive filelock with flock.
+          flock --exclusive --timeout 900 $LOCK_FILE \
+          singularity exec --containall docker://${docker} \
+          echo "Singularity pulled ${docker} successfully"
```

Comment:
You may want to mention that [this is from the Cromwell docs](https://cromwell.readthedocs.io/en/stable/tutorials/Containers/#job-schedulers) because it's quite the workaround.